src/components/CurriculumComponents/OverviewTab/OverviewTab.tsx

```diff
@@ -82,10 +88,11 @@ const OverviewTab: FC<OverviewTabProps> = (props: OverviewTabProps) => {
         ));
         return (
           <OakLI
-            $mb="space-between-sssx"
+            $mb={["space-between-xs"]}
+            $mr="space-between-none"
             key={`${firstItem.split(" ").join("-")}`}
           >
-            {createBullet(firstItem, i)}
+            <ULNoStyles>{createBullet(firstItem, i)}</ULNoStyles>
```

Comment:
I'll take a look.

Reply:
Sorted this now. I ended up refactoring it a little. createBullet was really just a new component (so I created that as a PrincipleBullet component) and then passed in the sub-bullets as children. The file can definitely be refactored further, but I think it will do for this ticket.